onyo/lib/commands.py

```diff
@@ -340,41 +340,44 @@ def onyo_edit(inventory: Inventory,
 
 
 def onyo_get(inventory: Inventory,
-             paths: Optional[list[Path]],
-             depth: int,
-             machine_readable: bool,
-             match: Optional[list[Callable[[dict], bool]]],
-             keys: Optional[list[str]],
+             paths: Optional[list[Path]] = None,
+             depth: int = 0,
+             machine_readable: Optional[bool] = False,
```

Comment:
This seems to be a mistaken notion of `Optional`.
It's a bool and because it comes with a default a caller doesn't need to pass anything. But that's not what `Optional` declares. `Optional` means it could be `None`. (It's equivalent to `bool | None`) 


Note, however, that NumPy's notion of 'optional' in the docstring is different and correct as is, I think. This is just about type annotation.